src/core/tests/frontend/paddle/test_models/gen_scripts/generate_elementwise_ops.py

```diff
@@ -193,5 +219,39 @@ def main():
     axis = 0
     elementwise_ops("4", data_x, data_y, axis, in_dtype)
 
+    # test for elementwise_floordiv, support int and int64
+    # paddle1.8 support axis = [0, x_last_dims]
+    # paddle2.x only support axis = -1
```

Comment:
Nice finding! Would you please share the links that shows the difference of attribute "axis" between Paddle versions?

Reply:
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/floor_divide_cn.html#floor-divide
https://github.com/PaddlePaddle/Paddle/blob/release/2.3/python/paddle/tensor/math.py#L458
this is paddle.floor_divide finally to call elementwise_floordiv op , and default axis = -1 

https://www.paddlepaddle.org.cn/documentation/docs/zh/1.8/api_cn/layers_cn/elementwise_floordiv_cn.html#elementwise-floordiv doc for paddle1.8